Sources/SymDiff/source/BoogieVerify.cs

```diff
@@ -970,6 +972,10 @@ public static int RunVerificationTask(VerificationTask vt, Program prog, out boo
 
                 return (SErrors == null ? 0 : SErrors.Count);
             }
+
+            // Delete temp file used for this verification task
+            File.Delete(vtFile);
```

Comment:
Maybe if we're running in debug mode we could print the name of this file, along with the names of the implementations being compared, and not delete it.